tests/integration/simulacron/test_endpoint.py

```diff
@@ -99,7 +100,8 @@ def test_custom_endpoint(self):
         cluster = Cluster(
             contact_points=[AddressEndPoint('127.0.0.1')],
             protocol_version=PROTOCOL_VERSION,
-            endpoint_factory=AddressEndPointFactory()
+            endpoint_factory=AddressEndPointFactory(),
+            compression=False,
```

Comment:
Simulacron barfs on compression, which I had enabled locally. It passes on CI because compression isn't installed.